### PR TITLE
Remove infinite scroll from tables, Add CSV download coming soon overlay

### PR DESCRIPTION
**Pull request by aadilhasan**
- Remove infinite scroll from token holders, and followers/followings and add an overlay with CSV download coming soon at the end of the table

### CODE REVIEW

File: src/Components/Filters/SnapshotFilter.tsx [hunk 1]
```diff
@@ -245,7 +245,11 @@ export function SnapshotFilter({
     }
 
     const filterValues: Partial<CachedQuery> = {
-      activeSnapshotInfo: getActiveSnapshotInfoString(snapshotData)
+      activeSnapshotInfo: getActiveSnapshotInfoString(snapshotData),
+      // clear active view and go back to list page
```
Comment (aadilhasan) [inline src/Components/Filters/SnapshotFilter.tsx:249]: @kaushalmeena please let me know if this can break anything else.

Comment (kaushalmeena) [reply src/Components/Filters/SnapshotFilter.tsx:249]: no it will not break snapshot flow

File: src/hooks/useGetCommonOwnersOfPoaps.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export function useGetCommonOwnersOfPoaps(eventIds: TokenAddress[]) {
     }
 
     if (tokens.length > 0) {
-      setPoaps(prev => [...prev, ...tokens]);
+      setPoaps(prev => [...prev, ...tokens].splice(0, LIMIT));
```
Comment (kaushalmeena) [inline src/hooks/useGetCommonOwnersOfPoaps.ts:87]: are we sure, we should be using splice here, why not slice?

Comment (aadilhasan) [reply src/hooks/useGetCommonOwnersOfPoaps.ts:87]: In our case it does not make any difference if we use splice or slice, as we are anyway creating a new array when doing spread .

**Review by kaushalmeena — APPROVED** [review]

**Review by kaushalmeena — APPROVED** [review]

**Pull request merged**